test/zdtm/transition/thread-bomb.c

```diff
@@ -34,7 +34,7 @@ static void *thread_fn(void *arg)
 
 int main(int argc, char **argv)
 {
-	int max_nr = 1024, i;
+	int max_nr = 768, i;
```

Comment:
We need to find why it fails with 1024 and add a comment that explains this.

Reply:
I also saw failures with 768. So this does not help at all. It is really strange. The following error can be seen during tests:
```
Program terminated with signal SIGABRT, Aborted.
#0  __GI_raise (sig=sig@entry=6) at ../sysdeps/unix/sysv/linux/raise.c:50
50	../sysdeps/unix/sysv/linux/raise.c: No such file or directory.
[Current thread is 1 (Thread 0x7f18252e8700 (LWP 3515))]
(gdb) #0  __GI_raise (sig=sig@entry=6) at ../sysdeps/unix/sysv/linux/raise.c:50
#1  0x00007f1851a37859 in __GI_abort () at abort.c:79
#2  0x00007f1851a37729 in __assert_fail_base (
    fmt=0x7f1851bcd588 "%s%s%s:%u: %s%sAssertion `%s' failed.\n%n", 
    assertion=0x7f1851c1c1cc "freesize < size", 
    file=0x7f1851c1c1a6 "allocatestack.c", line=389, function=<optimized out>)
    at assert.c:92
#3  0x00007f1851a48f36 in __GI___assert_fail (
    assertion=assertion@entry=0x7f1851c1c1cc "freesize < size", 
    file=file@entry=0x7f1851c1c1a6 "allocatestack.c", line=line@entry=389, 
    function=function@entry=0x7f1851c1c260 <__PRETTY_FUNCTION__.13152> "advise_stack_range") at assert.c:101
#4  0x00007f1851c0d8ce in advise_stack_range (guardsize=<optimized out>, 
    pd=139741679748864, size=<optimized out>, mem=0x7f18252d8000)
    at allocatestack.c:389
#5  start_thread (arg=<optimized out>) at pthread_create.c:567
#6  0x00007f1851b34293 in clone ()
    at ../sysdeps/unix/sysv/linux/x86_64/clone.S:95
```

But, I think, I am only seeing this with clang. Let's handle this PR as the last and try to get the other CI changes merged.